src/server/game/Guilds/Guild.cpp

```diff
@@ -1203,6 +1203,12 @@ bool Guild::Create(Player* pLeader, std::string const& name)
     _CreateDefaultGuildRanks(pLeaderSession->GetSessionDbLocaleIndex()); // Create default ranks
     bool ret = AddMember(m_leaderGuid, GR_GUILDMASTER);                  // Add guildmaster
 
+
```

Comment:
I aint gonna let you get away with adding too many blank lines.

Remove it :D

Reply:
hehe, removed!